test/javascript/components/dropdowns/Dropdown.test.js

```diff
@@ -7,7 +7,6 @@ import userEvent from '@testing-library/user-event'
 test('down arrow opens menu on first item', async () => {
```

Comment:
Would it make sense to add a test to verify that the ID is generated randomkly?

Reply:
Meh, it's more important that each item has a unique ID, than that they are random. I would test: each item has an ID and non of the ID is the same.